### DISCUSSION TITLE
Setting suggestions for keeping Video File Sizes to a minimum?

### DISCUSSION THREAD

**Discussion by OodiePasche** [ntsc-rs/ntsc-rs#112] [Q&A]
Personal use case: Had a 20 min, 121 mb video file turn into 3gb using the default rendering settings of H.264, 23 Quality, 5 encoding. Tried fidgeting with Quality + Encoding Speeds but I'm not quite understanding their impact other than it can make the file size wildly large without much visual change.

Any suggestions to keep it modestly reasonable without it adversely affecting the render? Struggling to find the sweet spot. 

Side note, wouldn't mind a future feature or setting that could quickly/automatically adjust settings, (low, med, high) or based on file size impact.

**Comment by Degamisu** [comment]
Hi. I would recommend setting the quality to much lower, also you should set the scanlines to something like 480 if possible. I usually use a quality of 10 - 15. It gets the job done with quality as well as filesize. let me know if this works for you!
